CHANGELOG.md

```diff
@@ -6,6 +6,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## Fixed
+
+ - Avoid corner-case in which the sequencer issues the same instruction multiple times when two units become non-ready at the same time
+
+## Added
+
+ - The sequencer can issue instructions to non-full units even if the other units are full
+
+## Changed
+
+ - The main sequencer issues instructions every time the target unit has a non-full instruction queue
+ - If the target unit is inside a lane, the instruction can be stalled if the target operand requesters are not ready
+ - New instruction enters the main sequencer with a token that marks the instruction as new
+ - The instructions are validated and counted when they enter the main sequencer
+ - A priority-pass mechanism allows an early issue of instructions stalled in the sequencer
```

Comment:
Are there five different changes, or are they part of the same fix?

Reply:
They belong to the same fix, but they are also different concepts. I squashed three of them anyway

Reply:
Okay. In the future, if that is the case, you can create subtopics in the list, instead of putting them as different topics. Makes the CHANGELOG a bit clearer.